PageIndex.md

```diff
@@ -19,9 +19,11 @@
 
 # ColumnIndex Layout to Support Page Skipping
 
-This document describes the format for column index pages in the Parquet
-footer. These pages contain statistics for DataPages and can be used to skip
-pages when scanning data in ordered and unordered columns.
+In Parquet, a "Page Index" is optional metadata for a
```

Comment:
I think there is also confusion that sometimes we say PageIndex, sometimes ColumnIndex. However, PageIndex is composed of ColumnIndex and OffsetIndex. Is it better to use PageIndex wherever possible? If agreed, we need to modify the title of this section as well.